Assets/MRTK/Examples/Experimental/SceneUnderstanding/Textures/DemoSpatialAwarenessAutoUpdate.png.meta

```diff
@@ -0,0 +1,110 @@
+fileFormatVersion: 2
```

Comment:
For a couple of the added icons, I wonder if we can reuse existing ones. For example, MRTK/Examples/Experimental/SceneUnderstanding/Textures/DemoSpatialAwarenessAutoUpdate.png looks like MRTK/SDK/StandardAssets/Textures/IconRefresh.png (and a couple others in that example textures folder)